src/server/pfs/server/driver.go

```diff
@@ -545,7 +545,9 @@ func (d *driver) createProjectInTransaction(ctx context.Context, txnCtx *txncont
 			"update project %s", req.GetProject().GetName())
 	}
 	// If auth is active, make caller the owner of this new project.
+	var username string
 	if whoAmI, err := txnCtx.WhoAmI(); err == nil {
+		username = whoAmI.GetUsername()
```

Comment:
In my variant of this PR I just made a Username function on txnCtx that doesn't return an error.  There is only one error that WhoAmI can return and it's pretty irrelevant for created_by work.